src/components/Homepage/Regions.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import Header from '../Navbar/Navbar';
+import Region from '../Detailspage/Region';
+import World from '../../Design/World_map.png';
+
+const Regions = () => {
+  const regionList = [
+    { region: 'Africa', country: 59 },
+    { region: 'Americas', country: 56 },
+    { region: 'Europe', country: 53 },
+    { region: 'Asia', country: 50 },
+    { region: 'Oceania', country: 27 },
+    { region: 'Antarctic', country: 5 }];
+  return (
+    <div>
+      <Header id="/" />
+      <div className="world-container">
+        <img src={World} alt="" className="world-map" />
+        <div className="centered">
+          <h3>Get Air Quality Data Where You Live.</h3>
+          <h5> Central Reporting Area.</h5>
+        </div>
+      </div>
+      <div className="row m-4">
+        {
+            regionList.map((element) => (
+              <Region
+                key={element.region}
+                region={element.region}
+                regionCountry={element.country}
+              />
+            ))
+        }
+      </div>
+    </div>
+  );
+};
```

Comment:
Your Home page should have a list of categories that could be filtered using given parameters, e.g. category name. Please try and add this feature so that your project meets the requirements. You could filter by alphabetical order the names of the regions 🤓

Reply:
@mateo951 I already filter the region inside ( region.js) Please can you explain me a little bit?
`const Region = (props) => {
  const { region, regionCountry } = props;
  const dispatch = useDispatch();
  let region1;
  if (region === 'Africa') region1 = Africa;
  else if (region === 'Asia') region1 = Asia;
  else if (region === 'Europe') region1 = Europe;
  else if (region === 'Oceania') region1 = Oceania;
  else if (region === 'Americas') region1 = Americas;
  else region1 = Antarctic;
  return (
    <div className="col-6 col-lg-4 col-md-4 border">
      <div className="region d-flex justify-content-between">
        <div className="d-flex flex-column">
          <img src={region1} alt="" className="m-2 img" />
          <div className="region-details">
            <p className="name">
              {region}
            </p>
            <p>
              {regionCountry}
              {' '}
              Countries
            </p>
          </div>
        </div>
        <div>
          <button
            key={region}
            type="button"
            className="btn mt-2"
            onClick={() => {
              dispatch(getCountries(region));
            }}
            to="/countries"
          >
            <NavLink to="/countries" className="link m-2 text-light">
              <FontAwesomeIcon icon={faCircleArrowRight} className="icon" />
            </NavLink>
          </button>
        </div>
      </div>
    </div>
  );
};`

Reply:
Yeah, sure! The requirement specifies that the **user** should be able to filter or order your regions in some way. For example, a toggle button that sorts out your regions in alphabetical order would be great!

Reply:
Done. A toggle button that sorts out the regions in alphabetical order.